receiver/azureblobreceiver/config.go

```diff
@@ -0,0 +1,39 @@
+// Copyright OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azureblobreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azureblobreceiver"
+
+import (
+	"go.opentelemetry.io/collector/config"
+)
+
+type Config struct {
```

Comment:
Can you open a follow up PR to document the fields directly in the Go struct? This is not urgent, but will be useful in the future for automatic configuration schema generation